src/test/java/vehicle/PostVehicleMotorCycleFieldRestrictions.java

```diff
@@ -0,0 +1,152 @@
+package vehicle;
+
+import data.GenericData;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import net.serenitybdd.junit.runners.SerenityParameterizedRunner;
+import net.thucydides.core.annotations.Steps;
+import net.thucydides.core.annotations.Title;
+import net.thucydides.core.annotations.WithTag;
+import net.thucydides.junit.annotations.TestData;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.http.HttpStatus;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import steps.VehicleTechnicalRecordsSteps;
+import util.BasePathFilter;
+import util.JsonPathAlteration;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+import static util.WriterReader.saveUtils;
+
+
+@RunWith(SerenityParameterizedRunner.class)
+public class PostVehicleMotorCycleFieldRestrictions {
```

Comment:
Please rename all files containing 'motorcycle' from '...MotorCycle...' to '...Motorcycle...' since motor and cycle actually form just one word

Reply:
Changes done. Please review